libs/saksbehandling-common/src/main/kotlin/behandling/Revurderingaarsak.kt

```diff
@@ -58,8 +58,8 @@ sealed class Utfall {
     }
 }
 
-// Disse må ha en oversettelse i frontend RevurderingAarsak.ts
-enum class RevurderingAarsak(
+// Disse må ha en oversettelse i frontend Revurderingaarsak.ts
+enum class Revurderingaarsak(
```

Comment:
Ikke fan av dette. Jeg vet denne typen endringer gir mer mening på norsk, men norsk særskriving gjør også hurtignavigering i IDE-er _vesentlig_ mer kronglete. 

Når hvert ord har stor forbokstav har man mulighet til å søke ved å _kun_ skrive forbokstavene. I dette tilfellet kan man skrive `RA` og få `RevurderingAarsak` høyt oppe på søkelisten. Ved å fjerne stor A må man skrive _hele_ ordet før man når relevant resultat.

Reply:
Jeg gjorde det siden noen hadde renamet kun i backend og i front

Men jeg vet ikke, det virker for meg som om vi kjører norsk i prosjektet foran hurtignavigering i IDE? :P

Reply:
Det jeg sier er ikke at det er kjempefarlig for meg men viktigst at det er likt i front og back filnavnene